docs/source/tutorials/memory_optimizations.rst

```diff
@@ -108,7 +108,7 @@ checkpointing, where all activations will either be recomputed later in the back
 
 To enable activation offloading, use the ``enable_activation_offloading`` config entry or flag
 in our lora finetuning single device recipe, e.g. ``enable_activation_offloading=True``. To allow
-usage of streams, make sure you are on a torch version later than PyTorch 2.5.0.dev20240907.
+usage of streams, make sure you are on a torch version equal to or later than PyTorch.
```

Comment:
```suggestion
usage of streams, make sure you are on a torch version equal to or later than PyTorch 2.5.0.
```

Reply:
bumping this

Reply:
I believe we can just revert the changes here to leave the file as-as since it's been updated in another PR @JP-sDEV